homeassistant/components/updater/__init__.py

```diff
@@ -95,15 +96,23 @@ def _load_uuid(hass, filename=UPDATER_UUID_FILE):
             newest = hass.components.hassio.get_homeassistant_version()
 
         # Validate version
+        update_available = False
         if StrictVersion(newest) > StrictVersion(current_version):
             _LOGGER.info("The latest available version is %s", newest)
-            hass.states.async_set(
-                ENTITY_ID, newest, {ATTR_FRIENDLY_NAME: 'Update Available',
-                                    ATTR_RELEASE_NOTES: releasenotes}
-            )
+            update_available = True
         elif StrictVersion(newest) == StrictVersion(current_version):
             _LOGGER.info(
                 "You are on the latest version (%s) of Home Assistant", newest)
+        elif StrictVersion(newest) < StrictVersion(current_version):
+            _LOGGER.debug(
+                "Local version is newer than the latest version (%s)", newest)
+
+        hass.states.async_set(
+            ENTITY_ID, 'on' if update_available else 'off', {
```

Comment:
Should it be a binary sensor platform now instead?

Reply:
Yes, from my point of view. 
What should be changed to make it a binary sensor?
this?
```
DOMAIN = 'updater'

ENTITY_ID = 'updater.updater'
```

Reply:
@MartinHjelmare Who can decide wether this should be a binary platform? If that speeds up the review, I'll make it a binary sensor platform.

Reply:
I suggest adding a binary sensor platform. The cloud binary sensor platform is a precedence.
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/cloud/binary_sensor.py

Reply:
I've committed a draft version. What do you think about it?